app/routes/surveys/components/SurveyEditor/SurveyEditor.js

```diff
@@ -66,15 +67,8 @@ function TemplateTypeDropdownItems({
             onClick={(e) => {
               e.preventDefault();
               e.stopPropagation();
-              if (
```

Comment:
Maybe add a wrapper around this as well, since `confirm` is removed?

Reply:
Hm, so I initially tried to wrap the list elements with a `ConfirmModalWithParent`, but discovered that those elements gets destroyed when you click on one of them, thereby no showing the Modal. Therefore, i moved the confirmation to the button opening the template-menu. ("bruk mal"). It is a bit less smooth, but it does its purpose ig. :thinking: